relays/ethereum/Cargo.toml

```
@@ -29,7 +29,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0.56"
 sp-bridge-eth-poa = { path = "../../primitives/ethereum-poa" }
 time = "0.2"
-web3 = { version = "0.12.0", default-features = false }
+web3 = { git = "https://github.com/tomusdrw/rust-web3", rev = "f8ed7137fefb1b1ba41e17b86078a6eddf3ca52b", default-features = false }
```

```suggestion
web3 = { version = "0.13.0", default-features = false }
```

Could you update to crates.io version? Just published it.